src/Tribe/Aggregator/Record/List_Table.php

```diff
@@ -100,22 +100,23 @@ public function prepare_items() {
 		if ( 'scheduled' === $this->tab->get_slug() && ! empty( $_REQUEST['s'] ) ) {
 			// nonce check if search form submitted.
 			$nonce = isset( $_POST['s'] ) && isset( $_POST['aggregator']['nonce'] ) ? sanitize_text_field( $_POST['aggregator']['nonce'] ) :  '';
-			if ( wp_verify_nonce( $nonce,  'aggregator_' . $this->tab->get_slug() . '_request'  ) ) {
+			if ( isset( $_GET['s'] ) || wp_verify_nonce( $nonce, 'aggregator_' . $this->tab->get_slug() . '_request' ) ) {
+				$search_term = filter_var( $_REQUEST['s'], FILTER_VALIDATE_URL ) ? esc_url_raw( $_REQUEST['s'] ) : sanitize_text_field( $_REQUEST['s'] );
```

Comment:
:warning: Equals sign not aligned with surrounding assignments; expected 8 spaces but found 1 space

<sub>hash: b4dd890561ff35d971cd0e8e5ca9678b</sub>